README.md

```diff
@@ -340,14 +346,14 @@ This is step-by-step how to install these **.files** for automatic setup Openbox
   </details>
   
 ### The step you are waiting for
-The final step is login into **openbox-session**, basically login from display manager you use such as lightdm, gdm, etc.
+The final step is login into **openbox-session**, basically login from your display manager like lightdm, gdm, etc.
 
-> Make sure the `sh` symlinks to `bash`, as it's uses [bashism](https://mywiki.wooledge.org/Bashism). **Why bash?** Simple, I'd say it's bloated and powerful.
+> Ensure `sh` is symlink to `bash`, as it's uses [bashism](https://mywiki.wooledge.org/Bashism). **Why bash?** Simple, it's **bloated** but **powerful**.
```

Comment:
Why do you invoke bash scripts with `sh`?

I guess bash is smart enough to prioritise the [shebang](https://en.wikipedia.org/wiki/Shebang_(Unix)) even when called as shell, but 

```sh
 $ man bash | less +/'the name sh'
...
       If  bash  is invoked with the name sh, it tries to mimic the startup behavior of historical versions
       of sh as closely as possible, while conforming to the POSIX standard as well.
...
```

---

You could've also added here some way to find it out, e.g. `realpath /bin/sh`.

Reply:
> Why do you invoke bash scripts with sh?

Actually, I can't seem to find `sh -c ` calling any of your scripts at all, although it used to be there in older commits sometimes.

It is quite obsolete now, then.

Reply:
That's actually called by `openbox-session` that invokes `sh` to execute their autostart script.

![2021-10-31-051120_1301x748_scrot](https://user-images.githubusercontent.com/53987136/139559748-b0a5396b-f777-4c3f-b72b-ed0709216d89.png)

![2021-10-31-051137_1301x748_scrot](https://user-images.githubusercontent.com/53987136/139559757-f2037233-b7e4-4f10-87c7-10299d759362.png)

![2021-10-31-051154_1301x748_scrot](https://user-images.githubusercontent.com/53987136/139559773-234078ca-5799-4f85-a799-ce69ad8c3265.png)

Reply:
> So it doesn't matter to switch system sh to bash rather than dash (even dash is a little faster).

Nay, bash is only some 2 milliseconds slower than shell, and it was not a question about speed at all.

This answers the question, however:
> That's actually called by openbox-session that invokes sh to execute their autostart script.

> I see some people has a problem when installing my dotfiles e.g blank, etc since can't execute autostart.

Reply:
Preparing POSIX-sh scrips for Joyful Desktop v4 🚀 🚀 🚀 

But at the moment I haven't prepared any UI changes at all, I'll let you know when the changes are final from version 3.2 to a bit cleaner I think.

Reply:
https://github.com/owl4ce/dotfiles/commit/e9f5677b1e02642a53ab24fa2888eac7a857b5d6

Reply:
Why did you end up appending `.sh` everywhere?

Reply:
> Why did you end up appending .sh everywhere?

Better file identification, in order to be used in rsync / find commands (debug). All configurations that are multiple for one package will renamed with suffix of each of their preferences, for example `dunstrc` -> `name.dunstrc`.